src/web-ui/src/public/Checkout.vue

```diff
@@ -15,151 +15,88 @@
-                <span>Tax (USD)</span>
-                <strong>${{ this.cartTaxRate.toFixed(2) }}</strong>
-              </li>
-              <li class="list-group-item d-flex justify-content-between">
-                <span>Shipping (USD)</span>
-                <strong>${{ this.cartShippingRate.toFixed(2) }}</strong>
-              </li>
-              <li class="list-group-item d-flex justify-content-between">
-                <span>Total (USD)</span>
-                <strong>${{ this.cartTotal.toFixed(2) }}</strong>
-              </li>            
-            </ul>
-            <form class="card p-2">
-              <div class="input-group">
-                <input type="text" class="form-control" v-model="order.promo_code" placeholder="Promo code">
-                <div class="input-group-append">
-                  <button type="submit" class="btn btn-secondary">Redeem</button>
-                </div>
+          <div class="col-md-4 order-md-2">
+            <div class="card p-1">
+              <div class="card-body">
+                <h4 class="d-flex justify-content-between align-items-center mb-3 card-title text-muted">
+                  Order Summary
+                </h4>
+                <ul class="list-group list-group-flush mb-3">
+                  <li class="list-group-item p-1 d-flex justify-content-between">
+                    <span>{{ this.cartQuantity }} item{{ this.cartQuantity === 1 ? '' : 's' }} in cart</span>
+                    <strong>${{ this.cartSubTotal.toFixed(2) }}</strong>
+                  </li>
+                  <li class="list-group-item p-1 d-flex justify-content-between border-top-0">
+                    <span>Shipping & handling:</span>
+                    <strong>${{ this.cartShippingRate.toFixed(2) }}</strong>
+                  </li>
+                  <li class="list-group-item p-1 d-flex justify-content-between">
+                    <span>Total before tax:</span>
+                    <strong>${{ this.cartTotalBeforeTax.toFixed(2) }}</strong>
+                  </li>
+                  <li class="list-group-item p-1 d-flex justify-content-between border-top-0">
+                    <span>Tax to be collected:</span>
+                    <strong>${{ this.cartTaxRate.toFixed(2) }}</strong>
+                  </li>
+                  <li class="list-group-item p-1 d-flex justify-content-between border-bottom-0">
+                    <span>Order total:</span>
+                    <strong>${{ this.cartTotal.toFixed(2) }}</strong>
+                  </li>
+                </ul>
+                <button class="btn btn-primary btn-lg btn-block" v-on:click="submitOrder">Place your Order</button>
               </div>
-            </form>
+            </div>
+
+            <button v-if="pinpointEnabled && user" v-on:click="triggerAbandonedCartEmail" class="abandoned-cart-btn btn btn-primary btn-lg m-4">
+              Trigger Abandoned Cart email
+            </button>
```

Comment:
Should this be on the cart view instead?

Reply:
+1

Reply:
Ah for this one I wasn't sure - the designs I saw had this on _both_ cart and checkout views so I brought it in additionally here

Reply:
According to the designs, it should be on both pages